generate.ts

```diff
@@ -732,13 +732,40 @@ function templatizeTree(locals: { [key: string]: any }, sourcePath: string, dest
     for (var i = 0; i < filesInDir.length; i++) {
         var filename = filesInDir[i];
         var file = sourcePath + "/" + filename;
+
+        var folderExcluded = false;
```

Comment:
I favor moving these into the isDirectory if/else below. That way, we only run the excludedFolders check on Directories and the excludedFiles check on files.